lib/automation/report_runner.rb

```diff
@@ -52,6 +51,7 @@ def initialize(
       )
         @sender = User.find_by(username: sender_username)
         @receivers = User.where(username: receivers)
+        @email_receivers = receivers.filter { |r| r.include? "@" }
```

Comment:
Maybe we should do a `receivers.filter!` before we use it to look up `User.where()` to save unnecessary querying.

Also to check which of the receivers are an email address, you can use `EmailAddressValidator.valid_value?(r)`